docs/versioned_docs/version-4.0.x/user-guides/upgrades.md

```diff
@@ -89,49 +101,23 @@ If the upgrade is from Local PV LVM or Local PV ZFS storage solution, additional
 
 This section describes the Replicated Storage upgrade from OpenEBS Umbrella chart 3.x to OpenEBS 4.0.0.
 
-1. Before upgrading to OpenEBS 4.0.0, write the existing helm values from your OpenEBS helm release to a local YAML file.
-
-```
-helm get values openebs -n openebs -a -o yaml > old-values.yaml 
-```
-
-2. Start the helm upgrade process with the new chart, i.e. 4.0.0 by using the below command:
+1. Start the helm upgrade process with the new chart, i.e. 4.0.0 by using the below command:
```

Comment:
We need the get values step for replicated as well.

Reply:
I've added it as a common setup before the split into local and replicated.